src/partials/movie-modal.html

```diff
@@ -59,8 +62,12 @@ <h3 class="modal__about-title">ABOUT</h3>
           </div>
         </div>
         <div class="modal__buttons">
-          <button class="modal__btn modal__btn--to-watched">ADD TO WATCHED</button>
-          <button class="modal__btn modal__btn--to-queue">ADD TO QUEUE</button>
+          <button id="modal-button-watched" class="modal__btn modal__btn--to-watched">
+            ADD TO WATCHED
+          </button>
+          <button id="modal-button-queue" class="modal__btn modal__btn--to-queue">
+            ADD TO QUEUE
+          </button>
         </div>
```

Comment:
Would you say it's less prone to error in a project if you'd use BEM to IDs too, wouldn't you? Most of them seem unnessary due to current class names.